chef/auth.py

```diff
@@ -71,10 +81,10 @@ def sign_request(key, http_method, path, body, host, timestamp, user_id):
         'x-ops-timestamp': timestamp,
         'x-ops-content-hash': hashed_body,
     }
-    
+
     # Create RSA signature
     req = canonical_request(http_method, path, hashed_body, timestamp, user_id)
     sig = _ruby_b64encode(key.private_encrypt(req))
     for i, line in enumerate(sig):
-        headers['x-ops-authorization-%s'%(i+1)] = line
+        headers['x-ops-authorization-%s' % (i + 1)] = line
```

Comment:
Please undo all these formatting-only changes. It makes the rest of the diff hard to read.